@utahdts/utah-design-system/react/components/containers/tabs/TabGroup.jsx

```diff
@@ -38,12 +50,39 @@ export function TabGroup({
           setTabGroupState((draftState) => { draftState.selectedTabId = tabId; });
         }
       },
+      navigateNext: () => {
+        const index = findCurrentTabIndex();
```

Comment:
modulo math may make this simpler?
`const nextIndex = (index + 1) % tabGroupState.tabs.length;`

Reply:
🤯